### DISCUSSION TITLE
Not seeing ollama when accessing remotely

### DISCUSSION THREAD

**Discussion by nntb** [open-webui/open-webui#3863] [General]
# Bug Report

## Description

**Bug Summary:**
When accessing web UI from an external URL the connection to ollama dosnt work. 

**Steps to Reproduce:**
Running Windows 11 install Ollama
Clone and setup open webui
Forward 8080 to a internet reachable url
Setup ollama
Add models 
Setup comfyui
Add connection to the main installation 
On an Internet connected device that's not connected to the local network browse to the URL that forwards to the 8080 
Login 
Models won't be available

**Expected Behavior:**
AI models and a connection to ollama should be available

**Actual Behavior:**
Not sure it used to work 

## Environment
v0.2.5
- **Open WebUI Version:** [e.g., 0.1.120]
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by nntb** [comment]
There is no docker logs and I don't know how to get the browser console logs

**Comment by sirajperson** [comment]
I have been having the same problem.

**Comment by nntb** [comment]
so i am guessing this is not a bug?

**Reply by nntb** [reply]
this issue magically resolved its self when I restarted the computer. no idea why it was not working.
